Manifest.txt

```diff
@@ -305,6 +305,7 @@ bundler/man/bundle-viz.1
 bundler/man/bundle.1
 bundler/man/gemfile.5
 bundler/man/index.txt
+checksum/bundler-2.2.1.gem.sha512
```

Comment:
Hei! As far as I understand, it can be useful to keep a list of checksums of released gems, but not sure if it makes sense to ship them with the released package? Probably not? :thinking:

Also, I guess we should provide checksums for the rubygems package too, although that case can't probably use this new task because rubygems is shipped in more formats than `.gem`. This was requested here: https://github.com/rubygems/rubygems/issues/2656.

Finally, we should probably go with a more automated and generic solution like what was proposed in https://github.com/rubygems/rubygems/issues/3379, but that one is much trickier since it requires us the change the lockfile format, so let's start with this :smile:.